### DISCUSSION TITLE
Untrawlable areas

### DISCUSSION THREAD

**Discussion by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#21] [General]
@Ned-Laman-NOAA @MargaretSiple-NOAA 
Let's start another discussion section here on untrawlable areas. I got lost with the assumptions about UT areas and I need to be caught up to speed again about them. 

1) Basic question but now I'm confused. How does a historical station become designated as untrawlable? If you can point me to documentation on that decision making somewhere that works too. 

2) Without the proper context, I feel weary about changing the boundaries of the UT areas when they're superimposed on the new stations. 

2a) What is the consequences/implications of respecting the boundaries of the UT areas (assuming that an UT station cannot be trawled) regardless of the depth strata to which it is assigned? E.g. if a historical station was an entire grid cell, 1-100 m, and designated as UT and it is now a station in the 75-150 m or 100-200 m stratum, is it too much of an assumption to also consider it all UT? 

2b) If a new station is partially untrawlable, what is the consequences of only considering the non-overlapping area of the new station in the station allocation process? We don't need stations to be either UT or UNK/Y in the allocation process because stations (those that are > 5 km2) are drawn proportional to their area.  

I feel like I'm a step behind on this discussion so thanks for helping me catch up.

**Comment by Ned-Laman-NOAA** [comment]
Responding to your questions one at a time:

Q1 in parts:

I'm parsing the first one about "historical station(s)" a little.

At this time, if a station is designated T, UT, or unknown in GOAGRID_GIS, we wouldn't change that "historical station" designation unless we drove the boat over it and made a determination. Thus, if by "historical station" you mean stations that have already been assessed one way or another, they only get changed when we inspect or sample them with the trawl net. Be aware that UT can change to T and vice versa. More on that below.

If you meant "Historically, how do stations get designated T or UT?," here's how that works. Note that all of the information to inform this process is initially stored in the MS Access Electronic Station Log at sea by the FPC and that information is translated later, typically be someone not the FPC, into the TRAWLABLE field in GOAGRID_GIS.
If you go to a station grid cell that is Unknown and all one stratum within the ~5km2 box and spend 2 hours canvassing that grid cell for trawlable ground without finding any, the grid cell can be designated permanently untrawlable IF the reasons for it being untrawlable are related to permanent features (e.g., bad bottom, pinnacles, too steep, too many boulders, etc.). If you go to a similar grid cell and determine it to be untrawlable because of weather or water conditions on that day (e.g., high currents come to mind), then regardless of time spent in the cell, you would leave the cell Unknown because you might be able to trawl it in different conditions. The search time part of this rule can be scaled appropriately to the portion of a ~5km2 within the assigned stratum (e.g., half the cell is in your stratum, an hour search can determine trawlability).
If you trawl a station and tear up the net, typically that station becomes untrawlable permanently. However, as FPC on the spot, you should have a conversation with the skipper about What if? What if we came back in different conditions? Would this station still be untrawlable? If there are reasons to believe you could tow it on a different day and have success, this would be noted in the Electronic Log and in your FPC notes and would be likely result in the station designation remaining unchanged after that attempt.
A successful tow in a grid cell (or portion thereof) renders that station Trawlable, even if there are other previously trawled sites where things when badly within that grid cell.
Where it gets tricky is if you've historically had a few good tows in a grid cell, but when you try to sample it this time or the last couple of times, you tear up or can't get a successful tow in. Here's where FPC notes and Electronic Station Log information is crucial to deciding whether the station becomes permanently untrawlable or remains available for station allocation in the future.

Q1 continued...documentation?
This process is not well documented in a single location. There are instructions for filling out the Electronic Station Log (G:\AI-GOA\Instructions&Procedures\Electronic station log) where many of the rules for determining Trawlability are codified to some extent. There is also a little bit of documentation around how to update GOA.GOAGRID_GIS here (G:\AI-GOA\Instructions&Procedures\GOAGRID Oracle database). However, much of the gray area decision making about assigning trawlability after the field season back here on the bench has resided in Paul and my heads as the ones who've been tasked with this in most recent years. I'm beginning to work with Bethany over last summer's station logs and plan to develop some documentation out of that process along with Paul's insights to try and capture how the decisions are made.

Q2

I agree that we need to be cautious about how we approach assigning T and UT with the new stratification. However, we need to do our best in a transparent and documented process to preserve the many hours of ship time represented by those designations and transfer them to the new stratification.

Q2a
It depends...In the Gulf, on the shelf, leaving a grid cell designated untrawlable in the past as untrawlable for the future despite a change to the depth stratification within that cell in the new design, is probably a fair assumption because shelf stations have historically been "tow the box not the depth" stations. That means, as long as you were on the shelf proper and not near a stratum boundary you could ignore the fact that you were towing outside the depths of the depth stratum you were assigned (note this did not apply to management zone strata). Therefore, if a grid cell on the shelf was designated UT, it should be UT because of permanent bottom features and should have been considered in totality of area when the search for trawlable ground was done.
Not so for formerly gully or slope strata or any stations near stratum boundaries. Historically, stations in these categories were "tow the depth not the box" stations meaning that if you got there and didn't have the depth under the boat that fell within the depth stratum assigned, you needed to go find an alternate station within the stratum assigned and with the proper depth and tow that. The original assigned station would remain unchanged for trawlability in this case since there was always an anticipation that we would redraw depth stratum boundaries at some point when we had better bathymetry. In terms of consequences, T/UT areal extent will change as we change designations as will the number of stations available to be allocated in the sampling design. I can't guess over the extent of the Gulf if this will produce a bias or will be a push when all said and done.

Q2b
Not sure I understand this one, but what I would say is that the designation of UT was primarily envisioned as a vessel time optimization step to keep us from sending vessels back to the same untrawlable spots year after year. Later people became aware that there was value to describing how much of the Gulf was untrawlable by our nets and we could do that from the efforts to classify stations under the stratified random design. Now that effort is supporting projects like SIRRCA interested in estimating abundance over untrawlable ground. The T/unknown designations are also a vessel time optimization tool because a skipper might be willing to "throw and go" at a previously good station saving us some time searching for a flat spot like would be necessitated by a designation of unknown.

Wow! That got long but hopefully was helpful.
